### ISSUE TITLE
[FR] Switches for types of audio feedback

### ISSUE THREAD

**Issue by bryanpedini** [MarlinFirmware/Marlin#22293]
### Is your feature request related to a problem? Please describe.

No, just a low priority nice to have in the future.

### Are you looking for hardware support?

Not related.

### Describe the feature you want

It would be nice to have a toggle and/or customization for buzzer/speaker events, such as `#define SOUND_ENABLE` in `configuration.h` and something like
```
#if ENABLED(SOUND_ENABLE) {
    #define SOUND_LCD_ENABLE // turn on sound feedback on LCD selections
    #define SOUND_ENDPRINT_ENABLE // turn on sound feedback when print job ends
    [...]
#endif
```
or something similar...

_Thanks in advance._

**Comment by thinkyhead** [comment]
Yes, it would be good to formalize the different kinds of audio feedback to some extent and to be able to make adjustments to those.
